### ISSUE TITLE
Muti-gpu search/retrain support ?

### ISSUE THREAD

**Issue by chaow94** [AutoML-4Paradigm/AutoSTR#5]
In papers, you mentioned that "All models are trained on 8 NVIDIA 2080 graphics cards. ".

When I run search code, I can use single GPU only, any suggestions?

**Comment by huizhang0110** [comment]
In the code,`nn.DataParallel` is leveraged to speed-up the searching and training procedure via multi-GPUs.
So you can reduce the number of GPUs directly by using a smaller `batch size`, of course `learning rate` should be tuned smaller.

**Comment by chaow94** [comment]
Thanks for your advice~
